src/auth/auth-domain.ts

```diff
@@ -9,7 +9,9 @@ const thirdPartyDomainSTSUrlMappings: { [domain: string]: string; } = {
   'myschoolautomation.com': 'https://sts-sso.myschoolautomation.com',
   'myschooldemo.com': 'https://sts-sso.myschooldemo.com',
   'myschooltraining.com': 'https://sts-sso.myschooltraining.com',
-  'smartaidforparents.com': 'https://account.smartaidforparents.com'
+  'smartaidforparents.com': 'https://account.smartaidforparents.com',
+  'smarttuition.com': 'https://account.smarttuition.com',
+  'smartaidforparents-com-stage.smarttuition.net': 'https://account.smartaidforparents-com-stage.smarttuition.net'
```

Comment:
TSLint enforces that object keys be in alphabetical order; the current order is causing the build to fail.